Mathlib/RingTheory/Binomial.lean

```diff
@@ -434,6 +438,10 @@ theorem choose_neg [NatPowAssoc R] (r : R) (n : ℕ) :
     ← descPochhammer_eq_factorial_smul_choose, descPochhammer_smeval_eq_ascPochhammer,
     show (-r - n + 1) = -(r + n - 1) by abel, ascPochhammer_smeval_neg_eq_descPochhammer]
 
+theorem choose_neg' [NatPowAssoc R] (r : R) (n : ℕ) :
+    choose (-r) n = Int.negOnePow n • multichoose r n := by
```

Comment:
I know this was copied from the previous theorem, but I wonder if `Int.negOnePow n` is needed over `(-1) ^ n`? `Int.negOnePow` is useful when the exponent is Int and pow is not well defined in general for negative exponent, but here `n` is Nat.

And it probably doesn't need to be smul either. If `(-1) ^ n` works, it will be inferred as `: R` so a plain `*` should work.

Reply:
Hmm, I see in #20192 it was argued that `Int.negOnePow` is better, so I am not sure

Reply:
I think this is the right spelling given that `choose_neg` is also spelled this way.